float64 temp
float64 pressure
float64 altitude
